pages/admin/request/create-replacement.tsx

```diff
@@ -65,15 +65,15 @@ export default function CreateReplacement() {
   });
 
   /** Reason for replacement section */
-  const [reasonForReplacement, setReasonForReplacement] = useState<ReasonForReplacementFormData>({
+  const reasonForReplacement: ReasonForReplacementFormData = {
```

Comment:
Can we rename this to `INITIAL_REASON_FOR_REPLACEMENT`? Also, we should move this object outside of this function component (it can still in this file, though) - that way, we avoid reinstantiating this object on every render.